src/libraries/Microsoft.Extensions.Configuration.Ini/src/IniStreamConfigurationProvider.cs

```diff
@@ -34,13 +34,13 @@ public class IniStreamConfigurationProvider : StreamConfigurationProvider
                 while (reader.Peek() != -1)
                 {
                     var rawLine = reader.ReadLine();
-                    var line = rawLine.Trim();
 
                     // Ignore blank lines
-                    if (string.IsNullOrWhiteSpace(line))
+                    if (string.IsNullOrWhiteSpace(rawLine))
                     {
                         continue;
                     }
+                    var line = rawLine.Trim();
```

Comment:
This is not a valid change.  Previously "    " would be skipped.  Now it'll blow up with an index out of range exception.